action.yaml

```diff
@@ -163,6 +163,8 @@ runs:
           # Create new deployment preview based on the deployment template file
           astro deployment create --deployment-file deployment-preview-template.yaml
 
+          # TODO: we need to add wait for deployment to be created, otherwise operation to copy airflow resources like connection is flaky if webserver is not up by then
```

Comment:
It's just something I noticed while writing tests, I am not sure if actual users also run into this from time to time, but it feels like something to fix down the line.

Reply:
The command has a `--wait` flag that might be good for this.